litmus-portal/frontend/src/components/ReliabilityScore/ResultTable.tsx

```diff
@@ -0,0 +1,190 @@
+import { Typography } from '@material-ui/core';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import React from 'react';
+import LinearProgressBar from '../ProgressBar/LinearProgressBar';
+import ToggleComponent from '../ToggleComponent';
+import useStyles from './styles';
+import InfoTooltip from '../InfoTooltip';
+import Center from '../../containers/layouts/Center';
+
+function createData(
+  name: string,
+  result: JSX.Element,
+  weight: number | number[],
+  points: number
+) {
+  return { name, result, weight, points };
+}
+const result = 9;
+const result1 = 6;
+const result2 = 3;
+const result3 = 2;
+
+interface ResultModalProps {
+  testValue: (number | number[])[];
+}
+
+const ResultTable: React.FC<ResultModalProps> = ({ testValue }) => {
+  const classes = useStyles();
+  const rows = [
```

Comment:
Use a loop, don't hardcode the no. of components

Reply:
I have used the map function later in the code. The data is static for the time being as we dont have the data or its format right now.